ddmrp/views/purchase_order_line_view.xml

```diff
@@ -40,6 +40,11 @@
         <field name="inherit_id" ref="purchase.purchase_order_line_search" />
         <field name="arch" type="xml">
             <filter name="hide_cancelled" position="after">
+                <filter
+                    name="hide_no_buffer"
+                    string="Hide lines without buffer"
```

Comment:
I think just "Buffered" will be a better name for the filter.

Reply:
Yes, changed it.